docs/snippets/common/init-command.yarn.js.mdx

```diff
@@ -0,0 +1,3 @@
+```shell
+yarn dlx storybook@next init
```

Comment:
@jonniebigodes Thanks for getting back to me on this. I've updated this init command and the upgrade command to use `latest` instead of `next`. 👍

Reply:
No need to thank whatsoever. My pleasure, one final thing, if you could also update the `storybook-upgrade-prerelease.yarn.js.mdx` to use the same structure (i.e., `latest´ instead of `next`), I'd appreciate it. As for now, we're recommending it to avoid any issues.